### PR TITLE
Nextcloud linux integration ```master```

### PR DESCRIPTION
**Pull request by 13Homer13**
Added screen shots and minor changes to guide flow.

### CODE REVIEW

File: site/source/guides/service-guides/nextcloud/nextcloud-setup/nextcloud-linux.rst [hunk 1]
```diff
@@ -16,7 +16,7 @@ Desktop Integrations
 --------------------
 Many Linux distributions ship with a Desktop Environment (DE) that supports Nextcloud account integration directly for use with their built-in calendars and other applications.  It is recommended to try these first for the best possible experience with your particular flavor of Linux.
 
-You will first need to :ref:`add your Root CA to your system<ca-linux>`.
+You will first need to :ref:`add your Root CA to your system<lan-linux>`.
```
Comment (kn0wmad) [inline site/source/guides/service-guides/nextcloud/nextcloud-setup/nextcloud-linux.rst:19]:
```suggestion
You will first need to :ref:`trust your Root CA<ca-linux>`.
```

`ca-linux` is the correct ref link for v035 and up docs

File: site/source/guides/service-guides/nextcloud/nextcloud-setup/nextcloud-linux.rst [hunk 2]
```diff
@@ -25,36 +25,117 @@ The following desktop environments support integrated account syncing, including
...
+    :width: 60%
+    :alt: open settings app
+    
+2. Go to Online Accounts and click on **Nextcloud**.
+
+  .. figure:: /_static/images/nextcloud/nextcloud-linux-2.png
+    :width: 60%
+    :alt: online accounts settings
+
+3. Type in the server path from **startOS > Nextcloud > Interfaces** and username and password from **startOS > Nextcloud > Properties**, and click **Connect**.
```
Comment (kn0wmad) [inline site/source/guides/service-guides/nextcloud/nextcloud-setup/nextcloud-linux.rst:42]:
```suggestion
3. Paste in the server path from **StartOS > Nextcloud > Interfaces** and the username and password from **StartOS > Nextcloud > Properties**, and click **Connect**.
```

File: site/source/guides/service-guides/nextcloud/nextcloud-setup/nextcloud-linux.rst [hunk 3]
```diff
@@ -25,36 +25,117 @@ The following desktop environments support integrated account syncing, including
...
+    :width: 60%
+    :alt: nextcloud account services
+
+5. Open the file manager, and you should see your NextCloud account in the side panel.
+
+  .. figure:: /_static/images/nextcloud/nextcloud-linux-5.png
+    :width: 60%
+    :alt: nextcloud in file manager
+
+To explore alternative distros beyond this guide, visit this link. `Linuxhint guide <https://linuxhint.com/linux_file_managers_nextcloud/>`_ and our `Nextcloud Master Thread <https://community.start9.com/t/nextcloud-master-thread/>`_.  Please share your feedback - it is very valuable to our community!
```
Comment (kn0wmad) [inline site/source/guides/service-guides/nextcloud/nextcloud-setup/nextcloud-linux.rst:60]:
```suggestion
To setup other Linux distributions, check out this `Linuxhint guide <https://linuxhint.com/linux_file_managers_nextcloud/>`_ and our `Nextcloud Master Thread <https://community.start9.com/t/nextcloud-master-thread/>`_.  Please share your feedback - it is very valuable to our community!
```

File: site/source/guides/service-guides/nextcloud/nextcloud-setup/nextcloud-linux.rst [hunk 4]
```diff
@@ -25,36 +25,117 @@ The following desktop environments support integrated account syncing, including
...
 
 Standalone Clients
 ------------------
-For if your desktop environment does not support account integrations, or if you just prefer to use a desktop client.
+For those that prefer to use a desktop client or your desktop environment does not support account integrations.
+
+.. note:: The desktop version of NextCloud doesn't have much of a user interface.  Once installed, it solely lives in the top right hand corner of the desktop in the navbar. When it's synced, the icon turns into a checkmark with a circle around it.
```
Comment (kn0wmad) [inline site/source/guides/service-guides/nextcloud/nextcloud-setup/nextcloud-linux.rst:66]:
```suggestion
.. note:: The desktop version of NextCloud doesn't have much of a user interface.  Once installed, it solely lives in the your system tray or navigation bar.  You can click on this icon to access the app.
```

File: site/source/guides/service-guides/nextcloud/nextcloud-setup/nextcloud-linux.rst [hunk 5]
```diff
@@ -25,36 +25,117 @@ The following desktop environments support integrated account syncing, including
...
-1. Open the client and click "Log In"
-2. From your server's Nextcloud Service page, go to "Interfaces" and copy the LAN address
-3. Enter your LAN address under "Server Address" and click "Next"
-4. You will be asked to Trust your server's certificate, which is safe to do as you generate and sign this during LAN Setup
-5. Tick the box for "Trust this certificate anyway" and click "Next"
-6. This will launch a page in your web browser, click "Log In" and then "Grant access" to link the desktop client. You can close this browser window afterwards
-7. Next, configure the local directory that you want to sync with Nextcloud. You may use the default or change it, and edit the sync settings to desired. When satisfied, click "Connect"
-8. Files will begin to sync immediately and you will see a green check when this is complete.
-9.  That's it! From this desktop client you will recieve notifications, control accounts and syncing, and quickly access your Apps' WebUI pages
+Make sure you have first set up :ref:`LAN access<lan-linux>`.
```
Comment (kn0wmad) [inline site/source/guides/service-guides/nextcloud/nextcloud-setup/nextcloud-linux.rst:74]:
```suggestion
Make sure you have first :ref:`trusted your Root CA<ca-linux>`.
```

File: site/source/guides/service-guides/nextcloud/nextcloud-setup/nextcloud-linux.rst [hunk 6]
```diff
@@ -25,36 +25,117 @@ The following desktop environments support integrated account syncing, including
...
+7.  That's it! From this desktop client you will recieve notifications, control accounts and syncing, and quickly access your Apps' WebUI pages
 
 Tor Setup
 .........
 You will first need to have the :ref:`Tor daemon running<tor-linux>`.
 
-1. On your desktop application. Click the account in the top left -> Settings, then in Settings, click Network, then "Specify proxy manually as" and "SOCKS5 proxy." Enter "127.0.0.1" for the Host and "9050" for the port.
-2. Close the Settings screen and click the account in the top left again, then "Add Account."
-3. On the following screen, click "Log in," then enter your Nextcloud Tor server address, which you can copy from the Nextcloud page on your Embassy -> Interfaces -> Tor. This must start with `http://` and end with `.onion`. Click Next.
+1. On your desktop application. Click the account in the top left > Settings, then click Network. Choose **Specify proxy manually as** and **SOCKS5 proxy**. Enter **127.0.0.1** for the Host and **9050** for the port.
```
Comment (kn0wmad) [inline site/source/guides/service-guides/nextcloud/nextcloud-setup/nextcloud-linux.rst:118]:
```suggestion
1. On your desktop application. Click the account avatar in the top left > Settings, then click Network. Choose **Specify proxy manually as** and **SOCKS5 proxy**. Enter **127.0.0.1** for the Host and **9050** for the port.
```

File: site/source/guides/service-guides/nextcloud/nextcloud-setup/nextcloud-linux.rst [hunk 7]
```diff
@@ -25,36 +25,117 @@ The following desktop environments support integrated account syncing, including
...
+    :width: 60%
+    :alt: nextcloud network settings
+
+2. Close the Settings screen and click the account in the top left again, then **Add Account**.
+
+  .. figure:: /_static/images/nextcloud/nextcloud-linux-tor-2.png
+    :width: 60%
+    :alt: nextcloud add account
+
+3. On the following screen, click **Log in your Nextcloud**, then enter your Nextcloud Tor server address, which you can copy from the Nextcloud page on your **startOS > Interfaces > Tor**. This must start with **`http://`** and end with **`.onion`**. Click **Next**.
```
Comment (kn0wmad) [inline site/source/guides/service-guides/nextcloud/nextcloud-setup/nextcloud-linux.rst:130]:
```suggestion
3. On the following screen, click **Log in your Nextcloud**, then enter your Nextcloud Tor server address, which you can copy from the Nextcloud page on your **StartOS > Interfaces > Tor**. This must start with **`http://`** and end with **`.onion`**. Click **Next**.
```

Comment (kn0wmad) [inline site/source/guides/service-guides/nextcloud/nextcloud-setup/nextcloud-linux.rst:130]: Did you test this with https by chance?  This is preferred if it works

File: site/source/guides/service-guides/nextcloud/nextcloud-setup/nextcloud-linux.rst [hunk 8]
```diff
@@ -25,36 +25,117 @@ The following desktop environments support integrated account syncing, including
...
+  .. figure:: /_static/images/nextcloud/nextcloud-linux-desktop-1.png
+    :width: 60%
+    :alt: nextcloud server path
+
+  .. figure:: /_static/images/nextcloud/nextcloud-linux-tor-3.png
+    :width: 60%
+    :alt: nextcloud add account
+
 4. This will launch your browser and prompt you to log in to your account. Log in and then grant access as we did for LAN.
 5. That's it! You may wish to set up some select folders for remote sync, but for large files, it is best to sync on LAN only, so you can "Skip folders configuration" on the resulting screen if you wish. Check your connection by clicking the newly created account in the client app.
```
Comment (kn0wmad) [inline site/source/guides/service-guides/nextcloud/nextcloud-setup/nextcloud-linux.rst:141]:
```suggestion
5. That's it! You can set up some select folders for remote sync, but for large files, it is best to sync on LAN only. Check your connection by clicking the newly created account in the client app.
```

**Review by kn0wmad — CHANGES_REQUESTED** [review]

File: site/source/guides/service-guides/nextcloud/nextcloud-setup/nextcloud-linux.rst (hunk 7 above)
Comment (13Homer13) [reply site/source/guides/service-guides/nextcloud/nextcloud-setup/nextcloud-linux.rst:130]: Will test it and correct.

**Review by kn0wmad — APPROVED** [review]

**Pull request merged**